client/sections.js

```diff
@@ -36,10 +36,21 @@ const sections = [
 		paths: [ '/me/account' ],
 		module: 'calypso/me/account',
 		group: 'me',
+		// /me/account/closed is enabled to logged out users for self-serve account restoration
+		// enableLoggedOut is needed as /me/account/closed is nested within /me/account
+		// redirectLoggedOut is now handled in client/me/account/index.js
+		enableLoggedOut: true,
+	},
+	{
+		name: 'account-closed',
+		paths: [ '/me/account/closed' ],
```

Comment:
We should also make sure this route doesn't get indexed by Google.

Something like this.

https://github.com/Automattic/wp-calypso/blob/761a6aa45f7572599ce7c34140d86c216f000691/client/landing/stepper/section.ts#L7